josue/app.test.js

```diff
@@ -0,0 +1,75 @@
+import { fireEvent, getByText, getById, getAllByText, getNodeText, getByLabelText, getByPlaceholderText} from '@testing-library/dom';
+import '@testing-library/jest-dom/extend-expect';
+import { JSDOM } from 'jsdom';
+import fs  from 'fs';
+import path  from 'path';
+import { async } from 'regenerator-runtime';
+
+const html = fs.readFileSync(path.resolve(__dirname,'./index.html'),'utf-8');
+
+let dom
+let window
+let document
+let container
+let scriptElement
+
+describe('Unit test login feature',()=>{
```

Comment:
Can you add a test to assert the error message appears when you fill the form with the wrong `confirm password` value?